.github/CONTRIBUTING.md

```diff
@@ -7,32 +7,39 @@ All commits whose authorship dates are not prior to `1420675200 +0000` are assum
 
 # Coding Standards
 
-### Absoloute Pathing
-Absoloute pathing has to be used for type, proc, and verb definitions. This is to make searching and reading easier.
+### Absolute Pathing
+Absolute pathing has to be used for type, proc, and verb definitions. This is to make searching and reading easier.
 
 An example of properly pathed code:
-```
+```DM
 /obj/item/device/cake
-    [cake code here]
+	[cake code here]
 
 /obj/item/device/cake/proc/eat_cake()
-    [proc code here]
+	[proc code here]
 ```
 
 An example of badly pathed code:
-```
+```DM
 /obj/item/device/cake
-    [cake code here]
+	[cake code here]
 
-    proc/eat_cake()
-        [proc code here]
+	proc/eat_cake()
+		[proc code here]
 ```
 
+### Initialize() over New()
+Since the implementation of the Stoned Master Controller, overrides of the `New()` proc have effectively become depracted in favour of `Initialize()` and `LateInitialize()`. In most cases, `Initialize()` is a drop-in replacement for `New()`, however, there are a few considerations to be taken into account when using this. Specifically, `Initialize()` must always return the parent proc's value. Either via the `. = ..()` semantics or with explicit `return ..()` statements.
+
+`LateInitialize()` is invoked as an asynchronous process whenever overridden.
```

Comment:
This isn't actually true. LI is called async during mapload, but immediately after Initialize() during normal operations.